slangpy/benchmarks/test_benchmark_param_array.py

```
@@ -0,0 +1,112 @@
+import pytest
+import numpy as np
+
+import slangpy as spy
+from slangpy.testing import helpers
+from slangpy.testing.benchmark import BenchmarkSlangFunction
+
+PARAM_SIZES = [16, 64, 256, 512]
+CALL_SHAPE = (1024, 1024)
+
+
+def _make_inputs(device: spy.Device, size: int):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_make_inputs`

(ANN202)